.github/workflows/update-api.yml

```diff
@@ -45,37 +34,22 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('**/pnpm-lock.yaml') }}
           restore-keys: ${{ runner.os }}-node-
 
-      - uses: pnpm/action-setup@v2.4.1
-        with:
-          version: '${{ steps.pnpm-version.outputs.version }}'
+      - uses: pnpm/action-setup@v4
 
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
           # use the version found in `.nvmrc`
           node-version-file: '.nvmrc'
           cache: 'pnpm'
 
-      # shows current pnpm version
-      - name: check pnpm version
-        run: pnpm -v
-
       - name: Install `node_modules`
         if: steps.cache-modules.outputs.cache-hit != 'true'
         run: pnpm install
 
       - name: Copy `.env.test` -> `.env`
         run: cp .env.test .env
 
-      - name: Cache build
-        uses: actions/cache@v3
-        id: cache-build
-        with:
-          path: .nuxt/
-          key: ${{ runner.os }}-build-${{ hashFiles('**/.nuxt') }}
-          restore-keys: ${{ runner.os }}-build-
```

Comment:
Is there a reason we're removing the caching step from the process?

Reply:
It didn't seem to work anymore, and not sure the benefit of caching builds specifically anyways as its good to have the build run in a roughly known good environment of CI